fsd_config/form_jsons/night_shelter/funding-required-ns.json

```diff
@@ -213,7 +213,8 @@
               "Financial year",
               "How it's been calculated",
               "Action"
-            ]
+            ],
+            "required": true
```

Comment:
We should change this for welsh forms too & cof, I can do this after we merge this and Rayhan confirms the fixes. 👍